app/src/main/res/layout/fragment_user_profile.xml

```diff
@@ -39,6 +39,7 @@
                     android:id="@+id/iv_text_drawable"
                     android:layout_height="84dp"
                     android:layout_width="84dp"
+                    android:layout_marginTop="10dp"
```

Comment:
why do we need `marginTop`, won't changing the gravity serve the purpose?

Reply:
@miPlodder as we can see in this image attached below, linearLayout that is parent of Imageview with `@+id/iv_text_drawable` has height set to `match parent` with `32dp` margin so it intersects with toolbar. so if i put only gravity as `center_vertical` user will see offset and more towards top instead of center as visible in image(smallest square), so i added offset of `marginTop="10dp"`.

![UserProfile](https://user-images.githubusercontent.com/70195106/103139787-d6422600-4705-11eb-87e9-a466f44f4fee.png)

Reply:
if you ought to compare, add ss for both the cases

Reply:
@miPlodder can you feel difference?

without marginTop
<img src="https://user-images.githubusercontent.com/70195106/103141004-2f18bb00-4714-11eb-8f9b-bc5ae654a687.jpeg" alt="ss" width="333">

with marginTop
<img src="https://user-images.githubusercontent.com/70195106/103141009-3fc93100-4714-11eb-9ed8-2a98db06bd15.jpeg" alt="ss" width="333">

Reply:
@miPlodder are there any changes i need to apply?

Reply:
remove additional margin, different device sizes will have a different orientation for it. Let's default it `center_vertical` only

Reply:
@miPlodder please review changes